src/neo/SmartContract/Native/ManagementContract.cs

```diff
@@ -18,6 +20,7 @@ public sealed class ManagementContract : NativeContract
 
         private const byte Prefix_NextAvailableId = 15;
         private const byte Prefix_Contract = 8;
+        private Lazy<byte[]> listContractsPrefix = new Lazy<byte[]>(() => new KeyBuilder(NativeContract.Management.Id, Prefix_Contract).ToArray());
```

Comment:
If you move the initialization to the constructor, then `Lazy<T>` is not needed.